### DISCUSSION TITLE
Phishing site listed by Google as the first result - raising awareness

### DISCUSSION THREAD

**Discussion by mihaitodor** [community/community#32765] [Other Feature Feedback, Questions, & Ideas] [answered]
I bumped into a high fidelity clone of GitHub last evening by searching for "benthos vader mihaitodor" on Google and thought I'd raise awareness. The site in question is: https://gh [dot] fakev [dot] cn but **DO NOT** navigate there. It literally shows up as the first result and the rest of the results have nothing to do with the repo that I'm looking for, namely https://github.com/mihaitodor/benthos-vader Someone went to a lot of trouble to build this and I don't get why Google isn't ranking GitHub above such sites or, ideally, detecting and removing them.

**Comment by airtower-luna** [comment, marked as answer]
You might want to report that to Google Safe Browsing: https://safebrowsing.google.com/safebrowsing/report_phish/

Obviously not quite as good as getting the site taken down (I hope the GitHub security team sees this!), but many browsers use the Safe Browsing list to block phishing sites. :slightly_smiling_face:

**Reply by mihaitodor** [reply]
Thank you! I did submit it there, but not sure how long it will take for them to act on it. In the meantime, I hope others see this and pay extra attention to the search result URLs.
